src/game/client/client_tf2vintage.vpc

```diff
@@ -260,6 +260,8 @@ $Project "Client (TF)"
 					$File	"tf\vgui\panels\tf_backgroundpanel.h"
 					$File	"tf\vgui\panels\tf_dialogpanelbase.cpp"
 					$File	"tf\vgui\panels\tf_dialogpanelbase.h"
+					$File	"tf\vgui\panels\tf_ItemSelectionpanel.cpp"
```

Comment:
Capitalization

Reply:
It isn't sensitive to a file with caps, or no?